packages/big-design/src/components/Button/Button.tsx

```diff
@@ -19,26 +19,18 @@ export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement>, Ma
   variant?: 'primary' | 'secondary' | 'subtle';
 }
 
-const RawButton: React.FC<ButtonProps & PrivateProps> = memo(({ forwardedRef, ...props }) => {
-  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
-    const { disabled, isLoading, onClick } = props;
-
-    if (onClick && !disabled && !isLoading) {
-      onClick(event);
-    }
-  };
+const LoadingSpinner = () => (
+  <LoadingSpinnerWrapper alignItems="center">
+    <ProgressCircle size="xxSmall" />
+  </LoadingSpinnerWrapper>
+);
 
-  const renderLoadingSpinner = () => {
```

Comment:
Not sure why we were defining this as part of the body of `RawButton`, there's no dependency on its props. Moved out.